### PR TITLE
Add "Create New Invoice" page

### LINKED ISSUE #10: Add "New Invoice" page

### PR DESCRIPTION
**Pull request by ashrivas8**
# Description

If added, this will create a page that lets the user add a new invoice. This page will be accessible from the `Add Invoice` button on the invoice list page.

Fixes #10 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/styles/common.css [hunk 1]
```diff
@@ -93,6 +94,7 @@ textarea {
   border-radius: 0.3em;
   border: 2px solid #ebf1fa;
   background-color: #ffffff;
+  min-width: 250px;
```
Comment (aromalanil) [inline src/styles/common.css:97]: Why is this required?

**Comment by aromalanil** [comment]
Add a screenshot for reference

File: src/styles/common.css (hunk 1 above)
Comment (ashrivas8) [reply src/styles/common.css:97]: Just because the text area element wasn't taking any of the empty surrounding space and looked weirdly small in the layout.

Before:

before

After:

after

File: src/pages/invoice/create.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+      <li class="bottom-nav-item">
+        <button onclick="showNavBar(event)"><i class="material-icons">menu</i></button>
+      </li>
+    </ul>
+  </nav>
+
+  <main class="content">
+    <header class="content__header">
+      <h1>New Invoice</h1>
+      <button class="btn btn-primary">
```
Comment (aromalanil) [inline src/pages/invoice/create.html]: Is there a `btn-primary` class?

Comment (ashrivas8) [reply src/pages/invoice/create.html]: It's not but it seems it's there in other files too. Good eye.

Comment (ashrivas8) [reply src/pages/invoice/create.html]: Done. Check this out [4a9bf89](https://github.com/ashrivas8/rz-billbook/pull/17/commits/4a9bf89a86ecdd51376f026e959360123c51497f) .

**Review by aromalanil — APPROVED** [review]
LGTM 🌷
